source/Personality.cpp

```diff
@@ -384,7 +398,7 @@ void Personality::UpdateConfusion(bool isFiring)
 Personality Personality::Defender()
 {
 	Personality defender;
-	defender.flags = STAYING | MARKED | HEROIC | UNCONSTRAINED | TARGET;
+	defender.flags = STAYING | MARKED | HUNTING | UNCONSTRAINED | TARGET;
```

Comment:
```suggestion
	defender.flags = STAYING | MARKED | HUNTING | UNCONSTRAINED | TARGET;
```
shouldnt defenders be both? daring and hunting

Reply:
Daring ships are now defenders too.